annotations/tests/test-core.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# FIXME: diff below ignores whitespace
+# FIXME: pass flags to AGG
+
+if [[ -z "$PASH_TOP" ]]; then
+    echo "Must set PASH_TOP" 1>&2
+    exit 1
+fi
+
+IN=$PASH_TOP/evaluation/scripts/input/1M.txt
+
+CMD="$1"
+FLG="$2"
+# AGG=$(grep aggregate ../$CMD.json | cut -d ':' -f '2' | awk '{$1=$1};1' | tr '"' ' ' | sed 's/,$//')
+AGG=$(cat ../$CMD.json | jq -r '.aggregate' | tr -d '$' | sed "s;PASH_TOP;$PASH_TOP;")
```

Comment:
I guess that is where you use the top level `aggregate`. I guess it is fine for now.